doc/manual/modules/performance/ref_available-tuned-plug-ins.adoc

```diff
@@ -84,6 +84,22 @@ It sets Aggressive Link Power Management (ALPM) to the value specified by the [o
 `mounts`::
 Enables or disables barriers for mounts according to the Boolean value of the [option]`disable_barriers` option. 
 
+`scheduler`::
+Allows tuning of scheduling priorities, processes/threads/IRQs affinities, and CPU cores isolation.
++
+The `default_irq_smp_affinity` parameter controls what *Tuned* will write to the `/proc/irq/default_smp_affinity`.
+The following values are supported:
++
+--
+** `calc` content of the `/proc/irq/default_smp_affinity` will be calculated from the `isolated_cores` parameter.
+Non-isolated cores are calculated as an inversion of the `isolated_cores`. Then the intersection of the non-isolated cores
+and the previous content of the `/proc/irq/default_smp_affinity` is written to the `/proc/irq/default_smp_affinity`.
+If the intersection is empty set, then just the non-isolated cores are written to the `/proc/irq/default_smp_affinity`.
+This behavior is the default if the parameter `default_irq_smp_affinity` is omitted.
+** `ignore` *Tuned* will not touch `/proc/irq/default_smp_affinity`.
+** cpulist like e.g. `1,3-4` the cpulist is unpacked and directly written to the `/proc/irq/default_smp_affinity`.
```

Comment:
s/cpulist like e.g./for a cpulist such as/

Reply:
It's list describing 'calc', 'ignore', and cpulist parameters and it's introduced by the sentence: 'The following values are supported:', that's why I didn't want to write 'for a cpulist' but just 'cpulist'.

Reply:
I reformatted the text to use asciidoc description, but I am not sure whether it looks better, nor whether the syntax I used is correct :)